R/get_genstate_vax.R

```diff
@@ -0,0 +1,46 @@
+#' Get general population state COVID vaccine data from CDC and Census 2020 18+ pop estimates
+#'
+#' @description Pulls the latest data from the CDC on COVID vaccinations initiated
+#' for adults in a state and then merges alongside a population number from the 18+ 2020
+#' Census population estimates for that state to use for rate calculation.
+#'
+#' @return a data frame with the following columns: State, Date,
+#' General.Initiated, General.Adult.Population
+#'
+#' @examples
+#' \dontrun{
```

Comment:
I wouldn't be opposed to running this example as part of the build! Never know when the CDC or Census might change links, etc. on us, and it's fast to run!